script.js

```diff
@@ -1,143 +1,138 @@
-const inbox = document.getElementById("inbox");
-const list = document.getElementById("list");
-let previousTaskElement = "";
-
-inbox.focus();
-function addTask() {
-  if (inbox.value === "") {
-    alert("Invalid entry. Try again.");
+const inbox = document.getElementById('inbox')
+const list = document.getElementById('list')
+let previousTaskElement = ''
+
+inbox.focus()
+
+function addTask () {
```

Comment:
"addTask" and "editTask" functions have too many responsibilities, we can separate dom-manipulation and data-manipulation as their own generic api, task for the next PR. I will explain more after the PR is merged (you can resolve this comment after you've read it)